packages/destination-actions/src/destinations/fullstory/identifyUserV2/index.ts

```diff
@@ -0,0 +1,71 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import { createUserRequestParams } from '../request-params'
+import { normalizePropertyNames } from '../vars'
+
+const action: ActionDefinition<Settings> = {
+  title: 'Identify User V2',
+  description: 'Sets user identity variables, creates user if it does not exist',
+  platform: 'cloud',
+  defaultSubscription: 'type = "identify"',
+  fields: {
```

Comment:
Do we no longer provide a prescriptive mapping for `anonymousId`? In the v1 variant we map `anonymousId` to `segmentAnonymousId_str` in FullStory.

Reply:
I wasn't sure on this one. I can keep anonymousId.

Reply:
Josh may have some thoughts, but that would maintain closer field parity with our existing v1 action.

Reply:
Please be sure to not add type suffixes though. `_str` gets treated as part of the field name in the V2 API.